carvis-web/src/server/routes/redisRoutes.js

```diff
@@ -0,0 +1,95 @@
+/* ======== REDIS SCHEMA =====
+places:
+---------
+hash name: place (string)
+keys:
+---
+userId:lat : int
+userId:lng : int
+userId:routable : string
+
+-- otherwise do getOwnPropertyNames to check another userId:data
+// NOTE: or is there a better way?
+
+===============================
+userdata:
+---------
+hash name: userId (alexa userId)
```

Comment:
does this mean we'd only be able to query by alexa userId? 

what if a user doesnt have one? in that case we dont check redis and instead do a db query by ph # or email? 

what if the user logs in via ph # - do we do a db query to get his alexa userId and then start looking up in the cache by that?

any possible way to have the hash name be an entire user object rather than just one field?